atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleDdlConfig.java

```diff
@@ -59,12 +60,29 @@ public boolean enableOracleEnterpriseFeatures() {
         return false;
     }
 
+    //TODO(hsaraogi): Create a ShrinkConfig class
     @Value.Default
     public boolean enableShrinkOnOracleStandardEdition() {
         return false;
     }
 
     @Value.Default
+    public long shrinkPauseSeconds() {
+        return 10;
+    }
+
+    @Value.Default
+    public long shrinkPauseOnFailureSeconds() {
+        return 60 * 30;
+    }
+
+    @Value.Default
+    public int shrinkConnectionTimeoutMillis() {
+        // This is set to 10 minutes as we saw socket timeouts with 5 minutes.
+        return 60 * 10 * 1000;
```

Comment:
nit: it's set to 10 minutes in a slightly strange way. `10 * 60 * 1000` would be more logical (same above)